src/pages/iou/request/step/IOURequestStepDistanceRate.tsx

```diff
@@ -65,6 +64,8 @@ function IOURequestStepDistanceRate({
 
     const currentRateID = TransactionUtils.getRateID(transaction) ?? '-1';
 
+    const rates = DistanceRequestUtils.getMileageRates(policy, false, currentRateID);
```

Comment:
I don't see the change to remove the Onyx keys. Is it already removed in another PR/refactor?